qldpc/codes.py

```diff
@@ -457,56 +516,55 @@ def get_distance(
         `CSSCode.get_distance_upper_bound` and `CSSCode.get_one_distance_upper_bound`.
 
         If `lower is False` and `upper is None`, compute an exact code distance with integer linear
-        programming.  Warning: this is an exponentially scaling (NP-complete) problem.
+        programming.  Warning: this is an exponentially difficult(NP-complete) problem.
```

Comment:
Perhaps rephrase it as — "This is an NP-complete problem, and it may take exponential time to execute."

Reply:
done